### DISCUSSION TITLE
CDN domains from WPO-Foundation/wptagent

### DISCUSSION THREAD

**Discussion by jatrost** [projectdiscovery/cdncheck#445] [Show and tell]
I am creating this issue mainly to share a resource.

This may be a useful addition. It contains 10's of common CDN domains.
https://github.com/WPO-Foundation/wptagent/blob/master/internal/optimization_checks.py#L67-L328

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
